### ISSUE TITLE
Color contrast insufficient in Description's Description Text

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7195]
**CKAN version**
https://demo.ckan.org/dataset/new

**Describe the bug**
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="#markdown"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Create Dataset - CKAN Demo - https://demo.ckan.org/dataset/new

Element path: a[href$="#markdown"]

Snippet: <a href="#markdown" title="" data-target="popover" data-content="<pre>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.29 (foreground color: #187794, background color: #ebebeb, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 107.0.1418.35

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by mgifford** [comment]
Yes. That's it.
